engine/table/src/main/java/io/deephaven/engine/table/impl/chunkfilter/StringChunkMatchFilterFactory.java

```diff
@@ -31,8 +31,8 @@ public boolean equalKey(String s, String s2) {
     private StringChunkMatchFilterFactory() {} // static use only
 
     @SuppressWarnings("rawtypes")
-    static ObjectChunkFilter makeCaseInsensitiveFilter(boolean invert, Object... values) {
-        if (invert) {
+    static ObjectChunkFilter makeCaseInsensitiveFilter(final MatchOptions matchOptions, final Object... values) {
```

Comment:
This isn't private; and we have a MatchOptions that could be indicating various case sensitivity so I think we need some documentation or some wrappers to make it not abusable.